python/xorbits/_mars/_version.py

```diff
@@ -10,12 +10,12 @@
 """Git implementation of _version.py."""
 
 import errno
+import functools
```

Comment:
Please do not include unrelated changes into this PR. Changes that fix the order of import statements should be an individual PR.